joplin/pages/event_page/fixtures/test_cases/city_location.py

```diff
@@ -0,0 +1,27 @@
+import os
+from pages.event_page.fixtures.helpers.create_fixture import create_fixture
+import pages.event_page.fixtures.helpers.components as components
+
+
+# A event page with a city_location
+def city_location():
```

Comment:
I'd go more verbose with the name here and do `event_with_city_location` or something. When `fixtures.city_location()` is called it seems like might be getting the location instead of the event